src/components/AccessibilityBody/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
```

Comment:
Should this be `mailTo`? Or is it clearer to leave it as mailto, since it's referring to the URI?

Reply:
Yeah, I was sort of debating that, and just went with the lowercase one as it's consistent with the attribute. I'm inclined to leave it as is.

Reply:
You could also just forgo creating the variable and just stick it in the `href` directly. It doesn't look like `mailto` is being used anywhere else.